plugins/rest/rest.go

```diff
@@ -354,15 +359,14 @@ func (c Client) Do(ctx context.Context, method, path string) (*http.Response, er
 	return resp, err
 }
 
-func withMaskedAuthorizationHeader(headers http.Header) http.Header {
-	authzHeader := headers.Get("Authorization")
-	if authzHeader != "" {
-		masked := make(http.Header)
-		for k, v := range headers {
+func withMaskedAuthorizationHeaders(headers http.Header) http.Header {
```

Comment:
suggestion: I think this new implementation is good, I'd name the function `withMaskedHeaders` however as the functionality is now more generic.

Reply:
I'll go ahead and make this change.  Thanks for pointing that out.